flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/runtime/aggregate/MinAggregate.scala

```diff
@@ -17,70 +17,96 @@
  */
 package org.apache.flink.api.table.runtime.aggregate
 
-import scala.reflect.runtime.universe._
+import org.apache.calcite.sql.`type`.SqlTypeName
+import org.apache.flink.api.table.Row
 
-abstract class MinAggregate[T: Numeric] extends Aggregate[T] {
+abstract  class MinAggregate[T: Numeric] extends Aggregate[T]{
 
-  var result: T = _
-  val numericResult = implicitly[Numeric[T]]
+  private val numeric = implicitly[Numeric[T]]
 
-  override def aggregate(value: Any): Unit = {
-    val input: T = value.asInstanceOf[T]
+  /**
+   * Initiate the partial aggregate value in Row.
+   * @param partial
+   */
+  override def initiate(partial: Row): Unit = {
+    partial.setField(aggOffsetInRow, numeric.zero)
```

Comment:
should initialize to the maximum value of the data type instead of `numeric.zero`.